system-x/services/ldap/src/main/java/software/tnb/ldap/service/Ldap.java

```diff
@@ -0,0 +1,63 @@
+package software.tnb.ldap.service;
+
+import software.tnb.common.account.NoAccount;
+import software.tnb.common.client.NoClient;
+import software.tnb.common.service.Service;
+import software.tnb.ldap.validation.LdapValidation;
+
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+public class Ldap extends Service<NoAccount, NoClient, LdapValidation> {
+
+    private static final Logger LOG = LoggerFactory.getLogger(Ldap.class);
+    private static final int LDAP_PORT = 389;
+
+    private static String ldapUsername = "cn=admin,dc=example,dc=org";
+    private static String ldapPassword = "admin";
```

Comment:
username + password should be in `account` class

Reply:
Done